Framework/Algorithms/test/AnnularRingAbsorptionTest.h

```diff
@@ -43,11 +43,11 @@ class AnnularRingAbsorptionTest : public CxxTest::TestSuite {
     MatrixWorkspace_sptr outWS = alg->getProperty("OutputWorkspace");
     TS_ASSERT(outWS);
 
-    const double delta(1e-08);
+    const double delta(1e-04);
```

Comment:
Is 1e-04 enough precision - that seems like a lot of variation between platforms?